content/en/docs/reference/terms/index.md

```diff
@@ -82,7 +82,7 @@ A FIDO2 [Discoverable Credential](#discoverable-credential) that is bound to a s
 
 ## Discoverable Credential
 
-A Discoverable Credential (previously known as a "resident credential" or "resident key") is a FIDO2/WebAuthn credential that is entirely stored in the authenticator (private key, credential ID, user handle, and other metadata). The [Relying Party (RP)](#relying-party-rp) also stores a copy of the _public_ key and credential ID
+A Discoverable Credential (previously known as a "resident credential" or "resident key") is a FIDO2/WebAuthn credential that can be used by a relying party without providing a user ID. As such, all parts of the credential are entirely stored in the authenticator (private key, credential ID, user handle, and other metadata).
```

Comment:
What if we emphasize a bit that it's the user driving the authentication by providing a discoverable credential to an RP?

```suggestion
A Discoverable Credential (previously known as a "resident credential" or "resident key") is a FIDO2/WebAuthn credential that can be used by a user to log into a relying party without initially providing a user ID. As such, all parts of the credential are entirely stored in the authenticator (private key, credential ID, user handle, and other metadata).
```

Otherwise this is a low-impact PR that's fine by me 🤷‍♂️

Reply:
even better, thanks!